api/controllers/MerchStoreController.ts

```diff
@@ -197,18 +204,57 @@ export class MerchStoreController {
     return originalOrder;
   }
 
-  @Patch('/order')
-  async editMerchOrder(@Body() fulfillOrderRequest: FulfillMerchOrderRequest,
+  @Patch('/order/:uuid/pickup')
+  async editMerchOrderPickup(@Params() params: UuidParam,
+    @Body() editOrderRequest: EditMerchOrderPickupRequest,
     @AuthenticatedUser() user: UserModel): Promise<EditMerchOrderResponse> {
-    if (!PermissionsService.canFulfillMerchOrders(user)) throw new ForbiddenError();
+    if (!PermissionsService.canAccessMerchStore(user)) throw new ForbiddenError();
```

Comment:
```suggestion
    if (!PermissionsService.canManageMerchOrders(user)) throw new ForbiddenError();
```

Reply:
Members can reschedule their pickup events, so they don't need store manager / store distributor permissions for this route